{{cookiecutter.repo_name}}/pyproject.toml

```diff
@@ -5,6 +5,7 @@ authors = [
     { name = "{{ cookiecutter.maintainer_name }}", email = "{{ cookiecutter.maintainer_email }}" },
 ]
 description = "{{cookiecutter.project_short_description}}"
+readme = "README.md"
```

Comment:
Is required to satisfy `long_description` in python package build. Twine requires this for packages uploaded to pypi.

Reply:
lgtm